SECURITY.md

```diff
@@ -0,0 +1,212 @@
+  faith.
+
+You are expected, as always, to comply with all applicable laws.
+
+If at any time you have concerns or are uncertain whether your security research
+is consistent with this policy, please submit a report through one of our
+[Official Channels](#reporting-a-vulnerability) before going any further.
+
+## Scope and Supported Versions
+
+We (the community) support the 4 most recent minor versions of Knative. For more
+details on what this means, see our
+[release principles](https://knative.dev/community/contributing/mechanics/release-versioning-principles/).
+
+While we encourage you to report vulnerabilities in any Knative repo, at HEAD or
+the tip of any release branch (release-X.Y), we commit to patching
+vulnerabilities only at HEAD and in currently supported release branches.
+
+Repos with releases have the releases listed in the Github UI, e.g.:
+
+- [serving](https://github.com/knative/serving/releases)
+- [eventing](https://github.com/knative/eventing/releases)
+
+Repos in Knative Sandbox that have no releases listed may have a higher bar for
+accepting vulnerability reports. No support is provided for archived repos.
+
+## Reporting a Vulnerability
+
+Please email the
+[Knative Security list](mailto:knative-security@googlegroups.com) with the
+following information:
+
+- the repo and version/branch that contains the vulnerability
+- a description of the type of vulnerability (e.g., buffer overflow)
+
+_Please avoid including any details which would allow reproduction of the issue
+at this stage._ Details will be requested subsequently.
+
+Someone from the list will get back to you within 3 business days. Accepted
+reports will be fixed within 90 calendar days via one or more releases for
+versions that are both affected and supported at that point.
+
+We ask that you keep reports confidential for the full 90 days, or until the
+security list members release you from the confidentiality, whichever comes
+first, as we coordinate disclosures with Knative partners who may need some time
+after the vulnerability is addressed in Knative in order to update their
+systems.
+
+### What's in a report?
+
+Initial reports via email should include the limited information described
+[above](#reporting-a-vulnerability). Followup details should include a
+description of the vulnerability, a proof-of-concept exploit demonstrating the
+vulnerability, and a statement of what this vulnerability puts at risk. Please
+send reports in English.
+
+The security list members may make a best-effort attempt to address reports that
+do not meet these standards.
+
+### Coordinated Vulnerability Disclosure
+
+Knative Security engages with partners, including some vendors of
+[Knative-based offerings](https://knative.dev/docs/knative-offerings/), to
+responsibly disclose newly discovered vulnerabilities.
+
+If you believe your organization should participate in Knative's Coordinated
```

Comment:
I'm curious about other downstreams that might not be vendors and hence not on this 'knative-offerings' list. Should there be a way to register to be notified of Knative vulnerabilities, or is the way to do this to ping KS to say, hi, I'm interested in being in the loop.

Reply:
Absolutely, that's exactly what this paragraph is intended for. Is there something missing from it that you think should be here?

Reply:
I just wanted to make sure how I read it was right (the '...or is the way to do this to ping KS...'), so all good :) Thanks!

Reply:
Yup, contacting Knative Security WG is how to get added. We'll want to have some sort of guidelines/policy/checklist for how to decide who to accept, but that doesn't need to be decided now nor added to this doc.